File: 2024_java_basics/09-for-loops/tasks/08-number-sequence.java
import java.util.Scanner;

public class NumberSequence {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int maxNum = Integer.MIN_VALUE;
        int minNum = Integer.MAX_VALUE;

        int rows = scanner.nextInt();
        
        for (int i = 0; i < rows; i++) {
            int number = scanner.nextInt();
            if (number > maxNum) {
                maxNum = number;
            }
            if (number < minNum) {
                minNum = number;
            }
        }
        
        System.out.println("Max number: " + maxNum);
        System.out.println("Min number: " + minNum);
        
        scanner.close();
    }
}

File: 2022_c#_basics/09-for-loop/08-number-sequence.cs
using System;

namespace _08_Number_sequence
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int maxNum = int.MinValue;
            int minNum = int.MaxValue;

            int rows = int.Parse(Console.ReadLine());

            for (int i = 0; i < rows; i++)
            {
                int number = int.Parse(Console.ReadLine());
                if (number > maxNum)
                {
                    maxNum = number;
                }

                if (number < minNum)
                {
                    minNum = number;
                }
            }

            Console.WriteLine($"Max number: {maxNum}");
            Console.WriteLine($"Min number: {minNum}");

        }
    }
}
